core/drivers/crypto/crypto_api/include/drvcrypt_mac.h

```diff
@@ -0,0 +1,30 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2020 NXP
+ *
+ * MAC interface calling the HW crypto driver.
+ */
+#ifndef __DRVCRYPT_MAC_H__
+#define __DRVCRYPT_MAC_H__
+
+#include <crypto/crypto_impl.h>
+#include <drvcrypt.h>
+#include <tee_api_types.h>
+
+/*
+ * Crypto library hmac driver allocation function prototype
```

Comment:
minor: prefer HMAC over hmac, here and there.

Reply:
Ok